README.md

```diff
@@ -74,6 +74,8 @@ in the config element of the workload configuration. In addition to those proper
 | 2                                    | durability-level          |
 | false                                | chunked                   |
 | stor/cosbench                        | manta-directory           |
+| 5242880                              | splitSize                 |
+| true                                 | multipart                 |
```

Comment:
I don't think we should default to true for MPU. It seems confusing to make this the default even though MPU is not yet available in the Triton Public Cloud.

Reply:
It is actually a documentation problem, for some reason I put true in the doc, where the code would say that it would default to false. I will update it.